doit/loader.py

```diff
@@ -27,7 +27,7 @@ def flat_generator(gen, gen_doc=''):
         if inspect.isgenerator(item):
             item_doc = item.gi_code.co_consts[0]
             for value, value_doc in flat_generator(item, item_doc):
-                yield value, value_doc
+                yield value, value_doc or gen_doc
```

Comment:
what is this change for? seems unrelated...

Reply:
ok. The commit message says why you added it.

This change does not break any test, but you also did not add any test... and I am not sure this is *always* the desired behaviour. I am reverting this change for now. 

So please create another issue to discuss what should be the expected behaviour. Can you show an example of why you make this change?

Reply:
I think it is for subtasks, in order to have the docstring of the parent generator. I can check later to give an example.